apprentice-react-o-shop/src/components/Main/Main.js

```diff
@@ -1,11 +1,29 @@
 import React from 'react';
+import { Route, Switch } from 'react-router';
```

Comment:
A convention we follow for imports is as follows: 
imports from 3rd party libraries such as react
*blank line*
imports from other libraries within our repository
*blank line* 
local imports such as your AdminOrders import

Reply:
Please apply this convention to across your project in your js files